examples/vm_scheduling/best_fit/launcher.py

```diff
@@ -0,0 +1,74 @@
+import io
+import os
+import random
+import timeit
+
+import yaml
+
+from maro.simulator import Env
+from maro.simulator.scenarios.vm_scheduling import DecisionPayload, PlaceAction, PostponeAction
+from maro.utils import convert_dottable
+
+CONFIG_PATH = os.path.join(os.path.split(os.path.realpath(__file__))[0], "config.yml")
+with io.open(CONFIG_PATH, "r") as in_file:
+    raw_config = yaml.safe_load(in_file)
+    config = convert_dottable(raw_config)
+
+
+if __name__ == "__main__":
+    start_time = timeit.default_timer()
+
+    env = Env(
+        scenario=config.env.scenario,
+        topology=config.env.topology,
+        start_tick=config.env.start_tick,
+        durations=config.env.durations,
+        snapshot_resolution=config.env.resolution
+    )
+
+    if config.env.seed is not None:
+        env.set_seed(config.env.seed)
+        random.seed(config.env.seed)
+
+    metrics: object = None
+    decision_event: DecisionPayload = None
+    is_done: bool = False
+    action: PlaceAction = None
+    metrics, decision_event, is_done = env.step(None)
+
+    while not is_done:
+        valid_pm_num: int = len(decision_event.valid_pms)
+        if valid_pm_num <= 0:
+            # No valid PM now, postpone.
+            action: PostponeAction = PostponeAction(
+                vm_id=decision_event.vm_id,
+                postpone_frequency=1
+            )
+        else:
+            # Get the capacity and allocated cores from snapshot.
+            valid_pm_info = env.snapshot_list["pms"][
+                env.tick : decision_event.valid_pms : ["cpu_cores_capacity", "cpu_cores_allocated"]
```

Comment:
Use env.frame_index instead env.tick is better, if the snapshot_resolution > 1, then env.tick will get incorrect snapshot.